book/contextMacros.ddoc

```diff
@@ -1 +1,2 @@
-DVER = 2.097
+COZUM_HTML = frontispiece.cozum.html
+DVER = 2.088
```

Comment:
Unrelated change

Reply:
Fixed. That file should really be added to .gitignore, since it's generated.

Reply:
see https://github.com/dlang/dlang.org/pull/3324